### PR TITLE
[Key Transparency] Add tile

### PR DESCRIPTION
**Pull request by pedrosousa**
### Summary

Add Key Transparency Auditor tile.

### CODE REVIEW

File: src/content/docs/key-transparency/monitor-the-auditor/index.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+```bash
+plexi [OPTIONS] <COMMAND>
+```
+
+### Configure your auditor remote
+
+`plexi` does not come with a default remote auditor, and you will need to choose your own. 
+
+You can do so either by passing `--remote-url=<REMOTE>` or setting the `PLEXI_REMOTE_URL` environment variable.
```
[the remark this answers is not in the document]
Comment (thibmeu) [reply src/content/docs/key-transparency/monitor-the-auditor/index.mdx:36]: this seems like a wrong comment

File: src/content/docs/key-transparency/api/auditor-information.mdx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+pcx_content_type: overview
+title: Auditor
+sidebar: 
+  order: 1
+---
+
+The Auditor is designed to sign epoch information, which includes the time at which the request is received by the Auditor, the epoch number, and the epoch digest. The Auditor serialises this information in binary using protobuf or bincode and checks whether the requested inclusion is valid, as in not seen, and incremental.
```
Comment (Oxyjun) [inline src/content/docs/key-transparency/api/auditor-information.mdx]: Is "as in not seen" a specific term? If so, might it be better to have these as bullet points?

File: src/content/docs/key-transparency/api/epochs.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+If a namespace is disabled, you receive the following error:
+
+```txt
+HTTP 400 Bad Request
+Namespace is disabled and read-only.
+```
+
+:::note
+Avoid publishing more than 100 epochs as reset might not work beyond this threshold.
```
Comment (Oxyjun) [inline src/content/docs/key-transparency/api/epochs.mdx]:
```suggestion
Avoid publishing more than 100 epochs, as reset might not work beyond this threshold.
```

File: src/content/docs/key-transparency/api/namespaces.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
+---
+pcx_content_type: overview
+title: Namespaces
+sidebar:
+  order: 2
+---
+
+The Cloudflare Key Transparency API is organized in namespaces, representing a log Cloudflare Auditor monitors. If you want to register a namespace, contact us.
```
Comment (Oxyjun) [inline src/content/docs/key-transparency/api/namespaces.mdx]:
```suggestion
The Cloudflare Key Transparency API is organized in namespaces, representing log Cloudflare Auditor monitors. If you want to register a namespace, contact us.
```

I'm not sure what `log Cloudflare Auditor monitors` are, so there's a chance this grammar change is not applicable.

File: src/content/docs/key-transparency/api/namespaces.mdx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+## Create a namespace
+
+The following fields are required when making a `POST` request:
+
+- `name`
+- `public`
+- `root`
+- `signature_version`: 
+  - 0x0001 is for [Protobuf serialisation](https://github.com/cloudflare/plexi/blob/main/plexi_core/src/proto/specs/types.proto) Ed25519 signature from the Auditor
```
Comment (Oxyjun) [inline src/content/docs/key-transparency/api/namespaces.mdx:18]:
```suggestion
  - 0x000 for [Protobuf serialisation](https://github.com/cloudflare/plexi/blob/main/plexi_core/src/proto/specs/types.proto) Ed25519 signature from the Auditor 
```

Comment (Oxyjun) [inline src/content/docs/key-transparency/api/namespaces.mdx]:
```suggestion
  - 0x0001 for [Protobuf serialisation](https://github.com/cloudflare/plexi/blob/main/plexi_core/src/proto/specs/types.proto) Ed25519 signature from the Auditor 
```

File: src/content/docs/key-transparency/api/namespaces.mdx [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+## Create a namespace
+
+The following fields are required when making a `POST` request:
+
+- `name`
+- `public`
+- `root`
+- `signature_version`: 
+  - 0x0001 is for [Protobuf serialisation](https://github.com/cloudflare/plexi/blob/main/plexi_core/src/proto/specs/types.proto) Ed25519 signature from the Auditor 
+  - 0x0002 is for [bincode serialisation](https://github.com/bincode-org/bincode/blob/trunk/docs/spec.md) E25519 serialisation by the Auditor
```
Comment (Oxyjun) [inline src/content/docs/key-transparency/api/namespaces.mdx]:
```suggestion
  - 0x0002 for [bincode serialisation](https://github.com/bincode-org/bincode/blob/trunk/docs/spec.md) E25519 serialisation by the Auditor
```

File: src/content/docs/key-transparency/api/auditor-information.mdx (hunk 2 above)
Comment (thibmeu) [reply src/content/docs/key-transparency/api/auditor-information.mdx]: bullet point would be ok. pushing that now

File: src/content/docs/key-transparency/api/namespaces.mdx (hunk 6 above)
Comment (thibmeu) [reply src/content/docs/key-transparency/api/namespaces.mdx]: resoved

File: src/content/docs/key-transparency/api/namespaces.mdx (hunk 5 above)
Comment (thibmeu) [reply src/content/docs/key-transparency/api/namespaces.mdx]: resolved

File: src/content/docs/key-transparency/api/namespaces.mdx (hunk 4 above)
Comment (thibmeu) [reply src/content/docs/key-transparency/api/namespaces.mdx]: updated to `each one representing a Log`

File: src/content/docs/key-transparency/api/epochs.mdx (hunk 3 above)
Comment (thibmeu) [reply src/content/docs/key-transparency/api/epochs.mdx]: updated

File: src/content/docs/key-transparency/api/auditor-information.mdx (hunk 2 above)
Comment (thibmeu) [reply src/content/docs/key-transparency/api/auditor-information.mdx]: updated with a link to constraints

File: src/content/docs/key-transparency/api/epochs.mdx [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+:::
+
+## Get an epoch
+
+```sh
+curl 'https://plexi.key-transparency.cloudflare.com/namespaces/{namespace}/audits/1'
+{
+  "namespace": "your.new.log.com",
+  "timestamp": 1717084639921,~
```
Comment (pedrosousa) [inline src/content/docs/key-transparency/api/epochs.mdx]:
```suggestion
  "timestamp": 1717084639921,
```

File: src/content/docs/key-transparency/index.mdx [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+Cloudflare's Key Transparency Auditor aims to secure the distribution of public keys for end-to-end encrypted (E2EE) messaging systems like [WhatsApp](https://engineering.fb.com/2023/04/13/security/whatsapp-key-transparency/). It achieves this by building a verifiable append-only data structure called a Log, similar to [Certificate Transparency](https://developer.mozilla.org/en-US/docs/Web/Security/Certificate_Transparency).
+
+Cloudflare acts as an auditor of Key Transparency Logs to ensure the transparency of end-to-end encrypted messaging public keys. Cloudflare provides an API for anyone to monitor the verification work we perform, and verify the state of its associated Logs locally.
+
+## Related products
+
+<RelatedProduct header="Certificate Transparency Monitoring" href="/ssl/edge-certificates/additional-options/certificate-transparency-monitoring/" product="ssl">
+
+Certificate Transparency (CT) Monitoring is an opt-in feature in public beta that aims at improving security by allowing you to double-check any SSL/TLS certificates issued for your domain.
```
Comment (Oxyjun) [inline src/content/docs/key-transparency/index.mdx]:
```suggestion
Certificate Transparency (CT) Monitoring is an opt-in feature in public beta that aims to improve security by allowing you to double-check any SSL/TLS certificates issued for your domain.
```

**Review by Oxyjun — APPROVED** [review]

File: src/content/docs/key-transparency/api/namespaces.mdx [hunk 9]
```diff
@@ -0,0 +1,86 @@
+---
+pcx_content_type: overview
+title: Namespaces
+sidebar:
+  order: 2
+---
+
+The Cloudflare Key Transparency API is organized in namespaces, each one representing a Log Cloudflare Auditor monitors. If you want to register a namespace, contact us.
```
Comment (pedrosousa) [inline src/content/docs/key-transparency/api/namespaces.mdx]:
```suggestion
The Cloudflare Key Transparency API is organized in namespaces, each one representing a Log monitored by Cloudflare Auditor. If you want to register a namespace, contact us.
```

**Pull request merged**